test/experiments/KiD_driver/parse_commandline.jl

```diff
@@ -15,11 +15,15 @@ function parse_commandline()
         "--moisture_choice"
         help = "Mositure model choice: EquilibriumMoisture, NonEquilibriumMoisture, CloudyMoisture"
         arg_type = String
-        default = "NonEquilibriumMoisture"
+        default = "EquilibriumMoisture"
         "--precipitation_choice"
         help = "Precipitation model choice: NoPrecipitation, Precipitation0M, Precipitation1M, Precipitation2M, CloudyPrecip"
         arg_type = String
         default = "Precipitation1M"
+        "--num_moments"
+        help = "Number of moments to use for CloudyPrecip (ignored otherwise)"
+        arg_type = Int
+        default = 7
```

Comment:
Should it be 6 by default? Maybe thats why the file does not get created and you get the netcdf error?